bootstrap_salt/fab_tasks.py

```diff
@@ -32,6 +32,21 @@ def aws(x):
 
 
 @task
+def environment(x):
+    env.environment = str(x).lower()
+
+
+@task
+def application(x):
+    env.application = str(x).lower()
+
+
+@task
+def config(x):
+    env.config = str(x).lower()
```

Comment:
Oh you want this because your fabfile.py doesn't include bootstrap_cfn, right?

Hmm.

These can be achived with `fab --set confg=foo.yml,application=my-app,aws=dev`

But maybe adding these here too is cleaner. Anyone have any thoughts?